src/components/pages/InstructorAddCourse/InstructorAddedCoursesList.js

```diff
@@ -8,17 +8,8 @@ const InstructorProgramList = props => {
 
   return (
     <div>
-      <h1
-        style={{
-          marginBottom: 10,
-          fontSize: 20,
-          display: 'flex',
-          justifyContent: 'center',
-        }}
-      >
-        Already Submitted Programs:
-      </h1>
-      <List style={{ display: 'flex', justifyContent: 'center' }}>
+      <h1 className="instructor-h1">Already Submitted Programs:</h1>
```

Comment:
```suggestion
      <h1 className="instructor-h1">Current Programs:</h1>
```

Reply:
Also, are we showing courses or programs?

Reply:
These renaming conventions are being done in a separate branch. It should all be courses except for the Program Type field in the form.